### DISCUSSION TITLE
对齐插图清单中的图题

### DISCUSSION THREAD

**Discussion by OZ111521** [tuna/thuthesis#911] [Q&A] [answered]
请问如何能够对齐插图清单和附表清单中标题呢？
![捕获](https://github.com/tuna/thuthesis/assets/153153121/884bc9f2-cba4-4e85-9b26-f520d93d2812)

**Comment by zepinglee** [comment, marked as answer]
虽然我赞同对齐更美观，但是研究生院提供的 Word 模板中是采取了固定间隔。

<img width="670" alt="Screenshot 2024-01-31 at 19 20 30" src="https://github.com/tuna/thuthesis/assets/12290822/f0bb5b98-cc56-4740-aa1d-58a17faf2198">
